cmake/option/option_noarch-tizen.cmake

```diff
@@ -22,7 +22,7 @@ SET(exec_prefix "${prefix}/bin")
 SET(includedir "${prefix}/include/${PROJECT_NAME}")
 SET(libdir "${prefix}/lib")
 SET(libname "${PROJECT_NAME}")
-CONFIGURE_FILE(${libname}.pc.in ${libname}.pc @ONLY)
+CONFIGURE_FILE(config/${libname}.pc.in ${libname}.pc @ONLY)
```

Comment:
Are these package config commands still needed in this file? This logic seems to be duplicated in higher-level cmake/libtuv.cmake .